CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 1.19.4 - 2025-09-04
+
```

Comment:
The definition of the number version is done during the release. Can you remove this change?
```suggestion
```